corehq/apps/accounting/invoice_pdf.py

```diff
@@ -412,10 +414,26 @@ def draw_footer(self):
         self.canvas.drawString(inches(5), inches(1.4),
                                "Thank you for using CommCare HQ.")
 
-        payment_description = """Payable by credit card, check, wire or ACH transfer.<br />
-For all payments, include "Invoice #" (displayed in top right corner).<br />
-<strong>Credit card payments</strong> are preferred and can be made online.<br />
-"""
+        payment_description = (
+            "Payment Options:<br />"
+            "<br />"
+            "<strong>Credit card payments</strong> are preferred and can be made online here: "
+            "<link href='%(payment_page)s' color='blue'>%(payment_page)s</link><br />"
+            "<br />"
+            "<strong>ACH or Wire:</strong> If you make payment via ACH or Wire, please make sure to email "
+            "<font color='blue'>%(invoicing_contact_email)s</font> "
+            "so that we can match your payment to the correct invoice.  Please include: "
+            "Invoice #, Project Space, and payment date in the email. <br />"
+        ) % {
+            'invoicing_contact_email': settings.INVOICING_CONTACT_EMAIL,
+            'payment_page': absolute_reverse(DomainBillingStatementsView.urlname, args=[self.project_name]),
+        }
+        payment_info = Paragraph('\n'.join([
+            payment_description,
+        ]), ParagraphStyle(''))
```

Comment:
It looks like this is `'\n'.join`ing a list with a single element, what's that about?